bin/setup-gocd-cli

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+from __future__ import annotations
+
+import argparse
+import hashlib
+import io
+import os.path
+import secrets
+import stat
+import subprocess
+import urllib.request
+
+URL = 'https://sentry-dev-infra-assets.storage.googleapis.com/gocd-085ab00-linux-amd64'  # noqa: E501
+SHA256 = '11d517c0c0058d1204294d01bfac987c0eaf9e37ba533ad54107b0949403321e'
```

Comment:
Can you add a comment on how you generated this value (i.e. `Run sha256sum gocd-...-linux-amd64`)

Reply:
I... copied it out of the shell script, lol